airbyte-integrations/bases/base-typing-deduping/src/test/java/io/airbyte/integrations/base/destination/typing_deduping/MockSqlGenerator.java

```diff
@@ -10,42 +10,42 @@
 class MockSqlGenerator implements SqlGenerator<String> {
 
   @Override
-  public StreamId buildStreamId(String namespace, String name, String rawNamespaceOverride) {
+  public StreamId buildStreamId(final String namespace, final String name, final String rawNamespaceOverride) {
     return null;
   }
 
   @Override
-  public ColumnId buildColumnId(String name) {
+  public ColumnId buildColumnId(final String name) {
     return null;
   }
 
   @Override
-  public String createTable(StreamConfig stream, String suffix) {
+  public String createTable(final StreamConfig stream, final String suffix, final boolean force) {
```

Comment:
I don't understand how this was successfully compiling previously. https://github.com/airbytehq/airbyte/pull/29637 should have had a broken build.